interactive-tests/default.rb

```diff
@@ -1,5 +1,15 @@
 # Test if model can be run at least for 0.5s without errors. Then it's reloaded and stopped (as some models
 # are automatically started in "onLoad" scripts, what would cause that screenshots would be always different).
+def visible
+  style = $test.driver.find_element(:css=>'.about-dialog').attribute("style")
+  if style.include?('block')
+    return true
+  else
+    return false
+  end
+
+end
+
```

Comment:
This function was added between comments and the code that comments were referring to. ;) But I guess it'd be nice to add this logic to test_api.rb as there's already similar function.